fiat-roles/src/main/java/com/netflix/spinnaker/fiat/providers/internal/Front50Service.java

```diff
@@ -50,40 +51,53 @@ public Front50Service(Front50Api front50Api) {
   public void afterPropertiesSet() throws Exception {
     try {
       // Initialize caches (also indicates service is healthy)
-      refreshApplications();
-      refreshServiceAccounts();
+      getAllApplicationPermissions();
+      getAllServiceAccounts();
     } catch (Exception e) {
       log.warn("Cache prime failed: ", e);
     }
   }
 
   public List<Application> getAllApplicationPermissions() {
-    return applicationCache.get();
+    return new SimpleJava8HystrixCommand<>(
+        GROUP_KEY,
+        "getAllApplicationPermissions",
+        () -> {
+          applicationCache.set(front50Api.getAllApplicationPermissions());
+          healthTracker.success();
```

Comment:
if this call fails when fiat first starts, it'll never go healthy?

Reply:
It will eventually since the `UserRoleSyncer` is scheduled to run every 10 minutes. This was actually the behavior until very recently. I was going to add some retries to the initial cache prime in a follow up but can add it as part of this as well!

Reply:
👍 

(I see `refreshCache()` back in here but nothing apparently calling it -- aside from that, this looks good to me)

We've ended up lowering the sync interval here from 10min to 2min (and actually feature flagged off the explicit sync'ing from `front50` to `fiat` due to it being an incident causing denial of service). 

We'll likely have to revisit that later bit but aren't yet creating applications with permissions initially set.

Reply:
Its scheduled to run every `"${fiat.front50RefreshMs:30000}`. But I also added to the initial call in `afterPropertiesSet()`